CHANGES/1748.misc

```diff
@@ -0,0 +1 @@
+Add CI tests for testing sync from console.redhat.com.
```

Comment:
```suggestion
Add CI tests for testing standalone mode syncing against insights mode
```
(since we are not attempting to hit console.redhat.com hostname)